src/components/Layout/Layout.tsx

```diff
@@ -6,40 +6,17 @@
  */
 
 import React, { PropsWithChildren } from 'react';
-// import { useStaticQuery, graphql } from "gatsby"
-
 import Header from '../Header';
-import { HorizontalNavigation, VerticalNavigation } from '../Navigation';
+import { HorizontalNavigation } from '../Navigation';
 import Footer from '../Footer';
 
-const Layout: React.FC<PropsWithChildren> = ({ children }) => {
-  // const data = useStaticQuery(graphql`
-  //   query SiteTitleQuery {
-  //     site {
-  //       siteMetadata {
-  //         title
-  //       }
-  //     }
-  //   }
-  // `)
-  return <>{children}</>;
-};
-
 const DefaultLayout: React.FC<PropsWithChildren> = ({ children }) => (
-  <Layout>
+  <>
     <Header />
     <main>{children}</main>
     <HorizontalNavigation />
     <Footer />
-  </Layout>
-);
-
-export const HomeLayout: React.FC<PropsWithChildren> = ({ children }) => (
-  <Layout>
-    <Header logoTag="h1" />
-    <VerticalNavigation />
-    <main>{children}</main>
-  </Layout>
```

Comment:
why did you do this?

Reply:
I wanted to challenge you to find a proper use for the layout composition. We could definitely preserve `<HomeLayout />` if you prefer, but I intended to highlight that `<Layout />` isn't in a position to demand its own component